recipes/mender-shell/debian-master/mender-shell.dirs

```diff
@@ -0,0 +1,3 @@
+/etc/mender
```

Comment:
Question:
Not `/etc/mender-shell/` ?

Reply:
The conf file is in `/etc/mender`, see https://github.com/mendersoftware/mender-shell/blob/master/Makefile#L57

I think it makes sense to have in the same directory all Mender related configuration